recipes/modsecurity.rb

```diff
@@ -33,13 +33,13 @@
 cache_path = Chef::Config[:file_cache_path]
 
 %w(
-  ModSecurity
-  owasp-modsecurity-crs
-  ModSecurity-nginx
+  SpiderLabs/ModSecurity
+  SpiderLabs/ModSecurity-nginx
+  coreruleset/coreruleset
```

Comment:
My proposal is that changes these repositories to attributes.

Reply:
Thanks, I fix it! https://github.com/linyows/nginx-cookbook/pull/11/commits/c09d24a6513026985ee8bd8fc22e5622670b7b44